Mtm50953 improvements for enterprise tenant documentation (#959)
SoftwareAG/c8y-docs: content/users-guide/enterprise-tenant-bundle/customization.md

|{host}|The value of this placeholder is "https://" + "&lt;&lt;tenantId&gt;&gt;" + "&lt;&lt;base-domain&gt;&gt;". For example, if "tenantId" is auto-generated, the host will be `https://t12345678.{{< domain-c8y >}}`.
|{tenant-domain}|This is the location in which a tenant can be accessed. It is equal to "https://" + "&lt;&lt;tenantDomainName&gt;&gt;". For example, {tenant-domain} can be `https://myTenant.{{< domain-c8y >}}`. In case of an {{< enterprise-tenant >}}, the {tenantDomain} placeholders can have different values. An example tenant domain is `https://myTenant.myhost.com`.
|{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".

> |{email}|Will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.

Reviewer: @dianaolives @Airdroplet Not sure about the sorting. Should it better be alphabetically? Or is there an order in which these placeholders appear?

|{username}|Will be replaced with the value of the username property specified in the user configuration, see [User options and settings](/users-guide/getting-started/#user-settings).
|{binaryId}|Will be replaced with the respective `binaryId` for the binary artefact to be used in the download link.
|{exportApi}|Will be replaced with the respective API in which the error occurred.
|{size}|Will be replaced with the storage usage percentage value.

{{< c8y-admon-info >}}

> Note that above mentioned placeholders might not be applicable to certain templates. While preparing content, be adviced by contextual information provided within user interface.

Reviewer: ```suggestion
The above mentioned placeholders might not be applicable to certain templates. While preparing content, note the information provided in the UI.
```

|{tenant-domain}|This is the location in which a tenant can be accessed. It is equal to "https://" + "&lt;&lt;tenantDomainName&gt;&gt;". For example, {tenant-domain} can be `https://myTenant.{{< domain-c8y >}}`. In case of an {{< enterprise-tenant >}}, the {tenantDomain} placeholders can have different values. An example tenant domain is `https://myTenant.myhost.com`.
|{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".
|{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.

> |{username}|

Reviewer: @Airdroplet According to the ticket from Daria, these placeholders are missing. If so, could you please provide the descriptions?

Reviewer 2: provided

|{tenant-domain}|This is the location in which a tenant can be accessed. It is equal to "https://" + "&lt;&lt;tenantDomainName&gt;&gt;". For example, {tenant-domain} can be `https://myTenant.{{< domain-c8y >}}`. In case of an {{< enterprise-tenant >}}, the {tenantDomain} placeholders can have different values. An example tenant domain is `https://myTenant.myhost.com`.
|{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".
|{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.

> |{username}|This placeholder will be replaced with the value of username property specified within user configuration, see: [User options and settings](/users-guide/getting-started/#user-settings).

Reviewer: ```suggestion
|{username}|Will be replaced with the value of the username property specified in the user configuration, see [User options and settings](/users-guide/getting-started/#user-settings).
```

> |{binaryId}|This placeholder will be replaced with respective binaryId for binary artifact for use in download link.

Reviewer: ```suggestion
|{binaryId}|Will be replaced with the respective `binaryId` for the binary artefact being used in the download link.
```

> |{exportApi}|This placeholder will be replaced with respective API on which error occured.

Reviewer: ```suggestion
|{exportApi}|Will be replaced with the respective API in which the error occurred.
```

> |{size}|This placeholder will be replaced with storage usage percentage value.

Reviewer: ```suggestion
|{size}|Will be replaced with the storage usage percentage value.
```
